portal-backend/depmap/cas/util.py

```diff
@@ -12,7 +12,7 @@ def get_storage_client() -> storage.Client:
     return client
 
 
-def get_private_datasets_bucket(client: storage.Client = None) -> storage.Bucket:
+def get_cas_bucket(client: storage.Client = None) -> storage.Bucket:
     if client is None:
         client = get_storage_client()
     bucket_name = current_app.config["CAS_BUCKET"]
```

Comment:
I'm not sure why this was named this before - it doesn't seem to be doing anything related to private datasets.